### DISCUSSION TITLE
Logging network traffic

### DISCUSSION THREAD

**Discussion by themg95** [RaspAP/raspap-webgui#1555] [Q&A]
Is there a way to log the network traffic of the devices connected to the AP?

**Comment by billz** [comment]
Yes, although it depends on what you mean by "traffic". **DHCP Server > Logging** gives you both DHCP and DNS requests from client devices. RaspAP does not do any invasive recording of URI requests or per-packet logging, however. For the latter, you'll need a tool like Wireshark.
